### PR TITLE
Add USD serie in block fee/reward charts

### PR DESCRIPTION
**Pull request by nymkappa**
Fixes https://github.com/mempool/mempool/issues/1520

This PR adds a new `USD` series to the block fees and rewards charts. It needs some polish.

Screen Shot 2022-07-09 at 6 27 17 PM
Screen Shot 2022-07-09 at 6 27 48 PM

### CODE REVIEW

**Comment by nymkappa** [comment]
Need to index block prices using the new `prices` table in the first place. Then this PR can be easily updated and reviewed.

**Comment by wiz** [comment]
please rebase and also change the colors back lol

**Comment by softsimon** [comment]
> ... and also change the colors back lol

I too prefer this old design. And maybe the USD graph could be in green which is the same color as fiat price / USD on the site.

Screen Shot 2022-07-10 at 21 15 21

**Comment by nymkappa** [comment]
Screen Shot 2022-07-11 at 9 21 24 AM
Screen Shot 2022-07-11 at 9 21 03 AM

File: backend/src/api/mining/mining.ts [hunk 1]
```diff
@@ -31,7 +33,7 @@ class Mining {
    */
   public async $getHistoricalBlockFees(interval: string | null = null): Promise<any> {
     return await BlocksRepository.$getHistoricalBlockFees(
-      this.getTimeRange(interval),
+      this.getTimeRange(interval, 5),
```
Comment (knorrium) [inline backend/src/api/mining/mining.ts:36]: @nymkappa what are you trying to do with this line?

I'm a bit confused given the current implementation of `getTimeRange`:

https://github.com/mempool/mempool/blob/a97675c5388e14edddfe87930112e75e7bd7f5c8/backend/src/api/mining/mining.ts#L531-L544

File: backend/src/repositories/BlocksRepository.ts [hunk 2]
```diff
@@ -288,6 +288,7 @@ class BlocksRepository {
       LIMIT 10`;
 
     try {
+      console.log(query, params);
```
Comment (knorrium) [inline backend/src/repositories/BlocksRepository.ts:291]: Use debug.log for this and add a TODO to remove it later

File: backend/src/api/mining/mining.ts (hunk 1 above)
Comment (nymkappa) [reply backend/src/api/mining/mining.ts:36]: It reduces the resolution x5 for the block fees charts to be more visually pleasing (less noisy)

This is what the chart look like without that scaling.

Screen Shot 2022-07-12 at 8 46 47 AM

VS now

Screen Shot 2022-07-12 at 8 47 35 AM

File: backend/src/repositories/BlocksRepository.ts (hunk 2 above)
Comment (nymkappa) [reply backend/src/repositories/BlocksRepository.ts:291]: Nah I just forgot it :), deleting

**Review by wiz — APPROVED** [review]
Tested ACK @ v2.5.0-dev [908635b3]

image

**Pull request merged**